pylot/operator_creator.py

```diff
@@ -25,6 +24,7 @@
 from pylot.perception.detection.lane_detection_operator import LaneDetectionOperator
 from pylot.perception.detection.obstacle_accuracy_operator import ObstacleAccuracyOperator
 from pylot.perception.detection.traffic_light_det_operator import TrafficLightDetOperator
+from pylot.perception.depth_estimation.depth_est_operator import DepthEstOperator
```

Comment:
Can you please wrap the import in a try except so that Pylot doesn't crash in case it doesn't have a copy of anynet. Like here: [https://github.com/erdos-project/pylot/blob/master/pylot/operator_creator.py#L21](https://github.com/erdos-project/pylot/blob/master/pylot/operator_creator.py#L21).